modes/es/aplicar.md

```
@@ -0,0 +1,114 @@
+5. **career-ops proof point**: incluir en "Información adicional" si existe ese campo
+
+**Campos específicos en formularios hispanohablantes habituales:**
+- **Pretensiones salariales (bruto anual)** -> Horquilla desde `profile.yml`, en EUR (o divisa local), con la indicación "negociable según el paquete global"
+- **Fecha de disponibilidad** -> Fecha realista teniendo en cuenta el preaviso (habitualmente 15 días a 3 meses)
+- **Permiso de trabajo / Nacionalidad** -> Honesto y conciso; para ciudadanos UE: "Sin necesidad de permiso de trabajo (ciudadano UE)"
+- **Idiomas** -> Niveles según el MCER (A1-C2)
+- **Movilidad** -> Especificar la zona geográfica aceptable y la frecuencia de desplazamiento
+
+**Formato de salida:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)